sonic-utilities-tests/system_health_test.py

```diff
@@ -0,0 +1,114 @@
+        assert result.output == expected
+        MockerConfig.ignore_devices.insert(0, "psu.voltage")
+        result = runner.invoke(show.cli.commands["system-health"].commands["detail"])
+        click.echo(result.output)
+        expected = """System status summary\n---------------------\nSystem status LED  red\n\nServices\tNot OK\n\t\ttelemetry is not Running\n\t\tsflowmgrd is not Running\nHardware\tOK\n\nSystem services and devices monitor list\n----------------------------------------\n\nName            Status    Type\n--------------  --------  ----------\ntelemetry       Not OK    Process\nsflowmgrd       Not OK    Process\nneighsyncd      OK        Process\nvrfmgrd         OK        Process\ndialout_client  OK        Process\nzebra           OK        Process\nrsyslog         OK        Process\nsnmpd           OK        Process\nredis_server    OK        Process\nintfmgrd        OK        Process\norchagent       OK        Process\nvxlanmgrd       OK        Process\nlldpd_monitor   OK        Process\nportsyncd       OK        Process\nvar-log         OK        Filesystem\nlldpmgrd        OK        Process\nsyncd           OK        Process\nsonic           OK        System\nbuffermgrd      OK        Process\nportmgrd        OK        Process\nstaticd         OK        Process\nvlanmgrd        OK        Process\nlldp_syncd      OK        Process\nbgpcfgd         OK        Process\nsnmp_subagent   OK        Process\nroot-overlay    OK        Filesystem\nfpmsyncd        OK        Process\nbgpd            OK        Process\nnbrmgrd         OK        Process\nfan12           OK        Fan\npsu_1_fan_1     OK        Fan\npsu_2_fan_1     OK        Fan\nfan11           OK        Fan\nfan10           OK        Fan\nPSU 2           OK        PSU\nASIC            OK        ASIC\nfan1            OK        Fan\nPSU 1           OK        PSU\nfan3            OK        Fan\nfan2            OK        Fan\nfan5            OK        Fan\nfan4            OK        Fan\nfan7            OK        Fan\nfan6            OK        Fan\nfan9            OK        Fan\nfan8            OK        Fan\n\nSystem services and devices ignore list\n---------------------------------------\n\nName         Status    Type\n-----------  --------  ------\npsu.voltage  Ignored   Device\n"""
+        assert result.output == expected
+        
+    @classmethod
+    def teardown_class(cls):
+        print("TEARDOWN")
+        os.environ["PATH"] = os.pathsep.join(os.environ["PATH"].split(os.pathsep)[:-1])
+        os.environ["UTILITIES_UNIT_TESTING"] = "0"
+        
```

Comment:
This is odd. There's an extra blank line at EOF, yet Git claims there is no newline at EOF. Can you fix this, please?